financial-connections/src/main/java/com/stripe/android/financialconnections/domain/SaveAccountToLink.kt

```diff
@@ -43,4 +49,38 @@ internal class SaveAccountToLink @Inject constructor(
             selectedAccounts = selectedAccounts
         )
     }
+        .updateCustomSuccessMessage(selectedAccounts.size)
+        .getOrThrow()
+
+    /**
+     * Update the custom success message in the [SuccessContentRepository] if the request was successful.
+     * If the request failed, update the custom success screen message to an error message
+     * (accounts were linked successfully, but not saved to Link).
+     */
+    private fun Result<FinancialConnectionsSessionManifest>.updateCustomSuccessMessage(
+        selectedAccounts: Int
+    ): Result<FinancialConnectionsSessionManifest> =
```

Comment:
To avoid duplicating logic, moved the success message to the domain layer (this use case gets called on a couple places)